food.go

```diff
@@ -182,11 +181,9 @@ func unarchiveOrCopy(src, dest, urlPath string) error {
 	}
 	defer in.Close()
 
-	if archive.IsArchivePath(src) {
-		return archive.Untar(in, dest, &archive.TarOptions{NoLchown: true})
-	} else if isZipPath(src) {
-		in.Close()
-		return unzip(src, dest)
+	err = archiver.Unarchive(src, dest)
```

Comment:
this line and the following line can be simplified to

```
if err := archiver.Unarchive(src, dest); err != nil {
```

Would you mind making that change?